src/avram/charms/int16_extensions.cr

```diff
@@ -19,7 +19,14 @@ struct Int16
       SuccessfulCast(Array(Int16)).new values
     end
 
+    def parse(value : Int32)
+      SuccessfulCast(Int16).new value.to_i16
+    rescue OverflowError
+      FailedCast.new
+    end
```

Comment:
This method is a duplicate of the one on line 35

Reply:
oh, good catch. I think one of them should probably be like Int64 and Int32...

Reply:
actually, no, because this is Int16 specific.